packages/suite/src/views/settings/coins/SettingsCoins.tsx

```diff
@@ -23,18 +24,9 @@ export const SettingsCoins = () => {
     const { anchorRef: anchorRefTestnetCrypto, shouldHighlight: shouldHighlightTestnetCrypto } =
         useAnchor(SettingsAnchor.TestnetCrypto);
 
-    const { device } = useDevice();
-
     return (
         <StyledSettingsLayout>
-            {device?.connected === false && (
```

Comment:
Is this deleted by accident?

Reply:
Yes, reintroduced in e1fe2661babdd65547300324d2557ddd0ac053fb